pam/main-cli.go

```diff
@@ -6,26 +6,27 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"github.com/msteinert/pam/v2"
-	"github.com/sirupsen/logrus"
-	"github.com/ubuntu/authd/internal/log"
 	"github.com/ubuntu/authd/pam/internal/pam_test"
 )
 
+func clearTerminal() {
+	// Use ANSI codes to clear the terminal.
+	// We may need this to prevent bubbletea window to hide our printed messages.
+	// See: https://en.wikipedia.org/wiki/ANSI_escape_code.
+	fmt.Print("\033[H\033[2J")
+}
+
 // Simulating pam on the CLI for manual testing.
 func main() {
-	log.SetLevel(log.DebugLevel)
-	f, err := os.OpenFile("/tmp/logdebug", os.O_CREATE|os.O_APPEND|os.O_RDWR, 0600)
-	if err != nil {
-		panic(err)
-	}
-	defer f.Close()
-	logrus.SetOutput(f)
+	logDir := os.Getenv("AUTHD_PAM_CLI_LOG_DIR")
 
 	module := &pamModule{}
 	mTx := pam_test.NewModuleTransactionDummy(pam.ConversationFunc(
 		func(style pam.Style, msg string) (string, error) {
+			clearTerminal()
```

Comment:
I still don't like this change, tbh. Clearing the stdout is a bad thing (especially because we rely on it to see whether our UI works or not), so I would just keep it as before and log the errors right below the UI.

Reply:
Yeah, I do agree... It was more or less something I figured out to get the whole error.

However, that commit should be removed and replaced by #236, whether in tests only or in all the cases (as I think is better).

Reply:
If you're fine with the rest of the MR though, I can drop the commits that cause changes to the golden files and I'll add to another MR once that one ready.

Reply:
Now that #236 got merged, can we rebase this one on main so we can see exactly how the diff will look like?

Reply:
test failure is unrelated, but I can't retry the job myself :(